include/rocksdb/table.h

```diff
@@ -555,7 +555,7 @@ struct PlainTableOptions {
 
   // @store_index_in_file: compute plain table index and bloom filter during
   //                       file building and store it in file. When reading
-  //                       file, index will be mmaped instead of recomputation.
+  //                       file, index will be mapped instead of recomputation.
```

Comment:
I think they intend to use mmap here?

Reply:
It is not actually mmap.  If this variable is true, the index is stored in the file and read back out (as a MetaBlock).  If the variable is false, no MetaBlock is written to the file and the index will be computed when the file is read.